Src/Lib/Lib.rc

```diff
@@ -27,18 +27,18 @@ LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
 // TEXTINCLUDE
 //
 
-1 TEXTINCLUDE 
```

Comment:
Please remove all these unnecessary white-space changes.

Reply:
The editor itself did that, I didn't. I'll do what I can though.

Reply:
Whitespace changes applied. Back to original with text changes for commit

Reply:
Thanks.
Now please squash those two commits with `Change homepage url in settings` message into one commit. So that it will contain just changes you meant to do.

Reply:
it's done

Reply:
You should end up with 2 commits - `Change homepage url in settings`, `ignore vscode generated files`.
You can easily rearrange commits using [interactive rebase](https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history) and then just force push (`git push -f`).

Reply:
Done